holoviews/tests/plotting/matplotlib/test_barplot.py

```diff
@@ -0,0 +1,145 @@
+import matplotlib.dates as mdates
+import numpy as np
+import pandas as pd
+from matplotlib.text import Text
+
+from holoviews.element import Bars
+
+from ...utils import LoggingComparisonTestCase
+from .test_plot import TestMPLPlot, mpl_renderer
+
+
+class TestBarPlot(LoggingComparisonTestCase, TestMPLPlot):
+
+    def test_bars_continuous_data_list_same_interval(self):
+        bars = Bars(([0, 1, 2], [10, 20, 30]))
+        plot = mpl_renderer.get_plot(bars)
+        ax = plot.handles["axis"]
+        np.testing.assert_almost_equal(ax.get_xlim(), (-0.4, 2.4))
+        assert ax.patches[0].get_width() == 0.8
+
+    def test_bars_continuous_data_list_diff_interval(self):
+        bars = Bars(([0, 3, 10], [10, 20, 30]))
+        plot = mpl_renderer.get_plot(bars)
+        ax = plot.handles["axis"]
+        np.testing.assert_almost_equal(ax.get_xlim(), (-1.2, 11.2))
+        assert ax.patches[0].get_width() == 2.4000000000000004
```

Comment:
```suggestion
        assert np.isclose(ax.patches[0].get_width(), 2.4)
```

Seems like it is just a floating point error. So lets treat it like that.